packages/lib-auth/src/models/oauth/OAuthStore.spec.js

```diff
@@ -0,0 +1,39 @@
+import OAuthStore from './OAuthStore'
+
+const CREDENTIALS = {
+  expiresAt: 123456789,
+  token: 'foobar'
+}
+
+const DEFAULT_CREDENTIALS = {
+  expiresAt: 0,
+  token: null
+}
+
+let oauthStore
+
+describe.only('Model > OAuthStore', function () {
+  beforeEach(function () {
+    oauthStore = OAuthStore.create()
+  })
+
+  it('should exist', function () {
+    expect(OAuthStore).to.not.equal(undefined)
+  })
+
+  it('should have an `credentials` property', function () {
+    expect(oauthStore.credentials).to.not.equal(undefined)
+  })
+
+  it('should have a `ui` property', function () {
+    expect(oauthStore.ui).to.not.equal(undefined)
+  })
+
+  it('should have a `tokenObject` property', function () {
+    oauthStore.credentials.set(CREDENTIALS)
+    expect(oauthStore.tokenObject.expiresAt).to.equal(CREDENTIALS.expiresAt)
+    expect(oauthStore.tokenObject.token).to.equal(CREDENTIALS.token)
+    oauthStore.credentials.reset()
+    expect(oauthStore.tokenObject).to.equal(null)
```

Comment:
This expectation can be `expect(null).to.be.null`

Reply:
Maybe the expectation test for the `tokenObject` to be null should be in its own it statement. Testing for null here doesn't really match up with `'it should have a tokenObject property'` Perhaps `it should not have a tokenObject property when reset`?

Reply:
The `tokenObject` getter is just a shortcut to the credentials for convenience. The function will still exist even if there are no credentials in the state, so it can be defined and return null. 

I have moved the expectation test though, good call.